.gitignore

```diff
@@ -114,3 +114,6 @@ site/
 
 # Virtual environments
 venv*
+
+# PyCharm
```

Comment:
I wonder how I didn't come across this one before. I probably added a global rule for this at some point...